src/website/views/components/button-group.pug

```diff
@@ -169,3 +169,191 @@ p.-text
         </div>
       </button>
     </div>
+
+h3 Sizes
```

Comment:
@davidojedacl I am missing the vertical sizes examples too. Can you add them?

Reply:
ok! for sure